explorer/deploy/webAppCreator.ts

```diff
@@ -172,9 +177,16 @@ class ResourceGroupStep extends WebAppCreatorStepBase {
             resourceGroups = results[0];
             locations = results[1];
             resourceGroups.forEach(rg => {
+                let location: string;
+                try {
+                    location = `(${locations.find(l => l.name.toLowerCase() === rg.location.toLowerCase()).displayName})`;
```

Comment:
>location [](start = 20, length = 8)

You might want to change the name to locationDisplayName or locationDescription.